src/runtime/audience-action-local-flow.ts

```diff
@@ -268,12 +216,48 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
   ) {
     this.rewardedReadyCalled_ = true;
     this.makeRewardedVisible_ = rewardedAd.makeRewardedVisible;
-
     const prompt = await this.prompt_;
-    prompt./*OK*/ innerHTML = REWARDED_AD_HTML;
 
-    // TODO: mhkawano - build UI.
-    // TODO: mhkawano - update when UX is done.
+    const isContribution =
+      this.params_.autoPromptType == AutoPromptType.CONTRIBUTION ||
+      this.params_.autoPromptType == AutoPromptType.CONTRIBUTION_LARGE;
+
+    // TODO: mhkawano - Provide internationalization.
+    // TODO: mhkawano - Fetch message and publication name from backend.
+    // TODO: mhkawnao - Support priority actions
+    // TODO: mhkawnao - Support premonetization
+    const publication = 'The Daily News';
```

Comment:
Any user-provided strings need to be HTML-escaped.

Reply:
Added as TODO for when we start getting the input.